src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,49 @@
+.nav {
+  &__list {
+    list-style: none;
+    padding: 0;
+    margin: 0;
```

Comment:
You are reseting these styles in `reset.scss` file. No need to write it twice. Please check everywhere🙏🏻